### ISSUE TITLE
Re-enable side list

### ISSUE THREAD

**Issue by MaximosPr** [tanujnotes/Olauncher#502]
Sorry for the wrong wording, it's just that when I did the recent update in playstore, the side list of letters when opening the apps menu (scrolling up) disappeared. It's a really useful feature especially if there are many apps installed.

**Comment by Setibs** [comment]
> Sorry for the wrong wording, it's just that when I did the recent update in playstore, the side list of letters when opening the apps menu (scrolling up) disappeared. It's a really useful feature especially if there are many apps installed.


Same here

**Comment by tanujnotes** [comment]
Too much crash and complexity. Will fix it, simplify and add it again later. Maybe just a thick scrollbar instead of letters list.

**Comment by MaximosPr** [comment]
> Maybe just a thick scrollbar instead of letters list. 
Yeah that would be nice. Same effect and muscle memory will make it efficient in no time.
